src/main/scala/esp/Implementation.scala

```diff
@@ -38,9 +38,16 @@ object ConfigIO {
 
 }
 
+object DmaSize {
+  private val enums = Enum(8)
+  val Seq(bytes, wordHalf, word, wordDouble, wordQuad, word8, word16, word32) = Enum(8)
```

Comment:
```suggestion
  val Seq(bytes, wordHalf, word, wordDouble, wordQuad, word8, word16, word32) = enums
```

Reply:
I see, sure, we can just commit this edit.